strabo/static/js/popup.js

```diff
@@ -88,7 +88,7 @@ while making sure that photoswipe is not opened twice.
 The delay is the second argument to window.setTimeout in units of milliseconds. This can be played with,
 but very small values like 1 do not work very well.
 */
-function set_flickety_click(){
+function set_flickity_click(){
     var timeout_sec = 0.1;
```

Comment:
Why do we include this function - is it to include a window timeout? I see that it is used later in this file but I'm confused by both the timeout_sec and SECS_PER_MILSEC variables.